tox.ini

```diff
@@ -34,6 +34,11 @@ extras =
 whitelist_externals =
     rm
     echo
+    git
+commands_pre =
+  # Retrieve possibly missing commits:
+  -git fetch --unshallow
+  -git fetch --tags
```

Comment:
This is necessary for `setuptools-scm` to calculate the distance to the closest tag.